clusters/ztp-policies/common/group-policies/group-smcx12-vse2.yaml

```
@@ -37,7 +43,133 @@ spec:
     #       node-role.kubernetes.io/worker: ""
     #     ptpEventConfig:
     #       storageType: "storage-class-http-events"
-
+    - fileName: PtpConfigGmWpc.yaml
+      policyName: "config-policy"
+      spec:
+        profile:
+        - name: "grandmaster"
+          phc2sysOpts: -r -u 0 -m -O -37 -N 8 -R 16 -s ens6f0 -n 24
+          ptp4lConf: |
+            [ens6f0]
+            masterOnly 1
```

Just a thought to discuss. Not urgent. Should we add all the ports in the WPC? Bc if used, all the ports will be used as master ports so they wont take any other role.

```suggestion
            masterOnly 1
            [ens6f1]
            masterOnly 1
            [ens6f2]
            masterOnly 1
            [ens6f3]
            masterOnly 1
```

---

I like this suggestion, thanks for making it here.